### PR TITLE
Updaing p2 repo with features related to G-Reg 5.2.0 release

### PR DESCRIPTION
**Pull request by thushara35**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -124,27 +133,35 @@
         <carbon.registry.version.4.4.9>4.4.9</carbon.registry.version.4.4.9>
         <carbon.registry.version.4.5.0>4.5.0</carbon.registry.version.4.5.0>
         <carbon.registry.version.4.5.2>4.5.2</carbon.registry.version.4.5.2>
+        <carbon.registry.version.4.5.3>4.5.3</carbon.registry.version.4.5.3>
+        <carbon.registry.version.4.5.4>4.5.4</carbon.registry.version.4.5.4>
         <carbon.rule.version.4.4.0>4.4.0</carbon.rule.version.4.4.0>
         <carbon.rule.version.4.4.1>4.4.1</carbon.rule.version.4.4.1>
         <carbon.social.version.2.3.4>2.3.4</carbon.social.version.2.3.4>
         <carbon.social.version.2.3.8>2.3.8</carbon.social.version.2.3.8>
+        <carbon.social.version.2.3.9>2.3.9</carbon.social.version.2.3.9>
```
Comment (callkalpa) [inline pom.xml]: This seems not being used, please check

File: pom.xml [hunk 2]
```diff
@@ -124,27 +133,35 @@
...
+        <carbon.social.version.2.3.10>2.3.10</carbon.social.version.2.3.10>
```
Comment (callkalpa) [inline pom.xml]: This seems not being used, please check

File: pom.xml [hunk 3]
```diff
@@ -157,23 +174,27 @@
         <process.feature.version.1.4.0>1.4.0</process.feature.version.1.4.0>
         <process.feature.version.1.4.2>1.4.2</process.feature.version.1.4.2>
         <process.feature.version.1.4.4>1.4.4</process.feature.version.1.4.4>
+        <process.feature.version.1.4.5>1.4.5</process.feature.version.1.4.5>
```
Comment (callkalpa) [inline pom.xml:172]: This seems not being used, please check

File: pom.xml [hunk 4]
```diff
@@ -1045,6 +1102,12 @@
                                     org.wso2.carbon.governance:org.wso2.carbon.governance.lifecycle.management.feature:${carbon.governance.version.4.6.1}
                                 </featureArtifactDef>
                                 <featureArtifactDef>
+                                    org.wso2.carbon.governance:org.wso2.carbon.governance.lifecycle.management.feature:${carbon.governance.version.4.6.2}
+                                </featureArtifactDef>
+                                <featureArtifactDef>
+                                org.wso2.carbon.governance:org.wso2.carbon.governance.lifecycle.management.feature:${carbon.governance.version.4.6.3}
```
Comment (callkalpa) [inline pom.xml]: Formatting issue

File: pom.xml (hunk 3 above)
Comment (thushara35) [inline pom.xml:172]: I have added the missing feature artifact Definition.

**Comment by thusithathilina** [comment]
Some features are missing in the PR. e,g org.wso2.carbon.registry:org.wso2.carbon.registry.social.feature 4.5.4. Hence closing the PR. Please send a PR with all the features

**Pull request closed without merging**